larq/layers.py

```diff
@@ -181,6 +182,7 @@ def __init__(
         kernel_size,
         strides=1,
         padding="valid",
+        pad_values=0.0,
```

Comment:
@larq/core how do you feel about `pad_values` vs `padding_values`?

Reply:
I'm fine with either, no particular preference.